cscs-checks/mch/collectives_halo.py

```diff
@@ -2,7 +2,7 @@
 import reframe.utility.sanity as sn
 
 
-class CollectivesBaseTest(rfm.RegressionTest):
+class CommunicationBaseTest(rfm.RegressionTest):
```

Comment:
Since it's not a test, but rather a base class, write it as follows:

```suggestion
class CommunicationTestBase(rfm.RegressionTest):
```